Packs/VirusTotal/README.md

```diff
@@ -1,5 +1,3 @@
-Note: Support for this pack will be moving to the partner around September 28, 2021.
```

Comment:
Please don't delete this line; instead:

Note: Support for this pack moved to the partner on Oct 1, 2021. Please contact the partner directly via the support link on the right.